spec/time_for_a_boolean_spec.rb

```diff
@@ -81,7 +82,7 @@
 
       it 'is false if the attribute date is in the future' do
         klass.time_for_a_boolean :attribute, :attribute_on
-        allow(object).to receive(:attribute_on).and_return(Date.current + 1)
+        allow(object).to receive(:attribute_on).and_return(Time.current + 1)
```

Comment:
Not that it will ever make a difference in practice, but the time (or even date) in the test should be specified in a proc.